app/assets/js/app/modules/print.js

```diff
@@ -0,0 +1,6 @@
+import domready from './domready'
+
+domready(() => {
+  Array.from(document.getElementsByClassName('btn-print'))
+    .forEach(el => el.addEventListener('click', () => window.print()))
+})
```

Comment:
Is this something that also needs to make its way into the pattern library?

Reply:
I'd guess that it wouldn't provide much value to standardise this sort of trivial script.

@saedwards @bennpearson what do you think?